### PR TITLE
modal popup page

### PR DESCRIPTION
**Pull request by xtrahuman**
# addition of modal pop-page
- I and my partner @mckent05 was involved in
- styling of modal page
- adding the content of the modal page with javascript
- added to javascript functions to open and close modal page when clicked
- array of content included as advised
- see live icon is visible
- .eslintrc.json folder is available
- added eslint to linters.yml

kindly see below a screenshot of see live icon showing in my browser
See-live
see-live2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                        <li><a href=${PageContent.skillsLink2}></a>${PageContent.skills2}</li>
+                        <li><a href=${PageContent.skillsLink3}></a>${PageContent.skills3}</li>
+                        <li><a href=${PageContent.skillsLink4}></a>${PageContent.skills4}</li>
+                    </ul>
+                </div>
+                </div>
+
+            </div>
+            <div class='view-btn'>
+            <button type="button" class="btn btn1">see live<span><img src="./asset/icon.png"></span></button>
```
[the remark this answers is not in the document]
Comment (xtrahuman) [reply index.js:53]: there is an icon image along with see live can you explain what you mean

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
```
[the remark this answers is not in the document]
Comment (xtrahuman) [reply .eslintrc.json:1]: this linter files and content is already in there, please can you be more detailed

[the remark this answers is not in the document]
Comment (GzimAsani) [reply .eslintrc.json:1]: I can see that your .github/workflows is different from the link that my college above mentioned. Please go back and check it https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml

File: index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (GzimAsani) [reply index.js:94]: Please check the picture. The icon is not showing
Screenshot from 2021-10-06 20-35-01

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi **Team**   
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good use of gitflow :+1: 

## Required Changes ♻️
- [ ] _Check the comments under the review for more information._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (xtrahuman) [reply .eslintrc.json:1]: added content to linters.yml

File: index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (xtrahuman) [reply index.js:94]: the icon is showing in my browsers, I just tested it with several browsers
kindly see screenshots below
See-live
see-live2

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         

## Thank you for implementing all the project requirements, you've done a great job :trophy:  . There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**